Documentation/install/system_requirements.rst

```diff
@@ -289,6 +289,19 @@ ICMP 8/0                 egress          ``worker-sg`` (self) health checks
           Routing mode, you can condense all rules into ingress/egress ANY
           port/protocol to/from self.
 
+The following ports should also be available on each node:
+
+======================== ==========  ==========================================
+Port Range / Protocol    Interface   Description
+======================== ==========  ==========================================
+4240/tcp                 pod IP      cluster health checks (``cilium-health``)
```

Comment:
This port is used for the host and the cilium-health endpoint IP (note, this is *not* a pod).

Reply:
With the `Interface` column omitted for now, I think this doesn't need any change.